sw/airborne/modules/nav/nav_rotorcraft_hybrid.c

```diff
@@ -35,6 +35,12 @@
 #define NAV_MAX_SPEED (GUIDANCE_INDI_MAX_AIRSPEED + GUIDANCE_INDI_NAV_SPEED_MARGIN)
 float nav_max_speed = NAV_MAX_SPEED;
 
+// Max ground speed in hover mode (goto/stay)
+#ifndef GUIDANCE_INDI_HOVER_SPEED
+#define GUIDANCE_INDI_HOVER_SPEED 5.f
```

Comment:
```suggestion
#define GUIDANCE_INDI_HOVER_SPEED NAV_MAX_SPEED
```
I'm not fully convinced with the name 'hover_speed'.